website/docs/checkaction.md

```diff
@@ -9,10 +10,12 @@ checkaction (value)
 `value` is one of:
 
 * `Default`: needs documentation
-* `D`: needs documentation
-* `C`: needs documentation
-* `Halt`: needs documentation
-* `Context`: needs documentation
+* `D`: call D assert on failure
+* `C`: call C assert on failure
+* `Halt`: cause program halt on failure
+* `Context`: call D assert with the error context on failure
+
+See https://dlang.org/library/dmd/globals/checkaction.html
 
 ## Applies To ###
 
```

Comment:
You need to align the example, because it is not valid now. Use: `checkaction "Halt"`